packages/translated-components/src/index.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react'
 import {Broadcast, Subscriber} from 'react-broadcast'
 import reduce from 'lodash/reduce'
+import transform from 'lodash/transform'
```

Comment:
Ta - removing. However, Ben & I are working on some code that might mean this whole PR is unnecessary :P